chapters/en/slides/module5/module5_05.md

```diff
@@ -1,3 +1,241 @@
+
+
+
+---
+
+# Giving collaborators access to your project
+
+<center>
+
+<img src='/module5/add-collab-2.png' width="80%" alt="404 image" />
+
+<center/>
+
+Notes: Then click “Manage access” and type in the collaborator’s GitHub username or email, and select their name when it appears.
+
+After this, you should see your newly added collaborator listed under the “Manage access” tab. They should receive an email invitation to join the GitHub repository as a collaborator. **They need to accept this invitation to enable write access**.
+
+
+---
+
+# Branch protection rules
+
+Under "Branch name pattern", type the branch name or pattern you want to protect. 
+
+<center>
+
+<img src='/module5/add-branch-protection-rule.png' width="80%" alt="404 image" />
+
+<center/>
+
+Notes: Now that all the collaborators have access they can clone the repository and create a branch using a meaningful name with the feature they are willing to work on. If some of the collaborators push to `main` without socializing it to the other team members first could create merging problems. To avoid that risk, you can set a [branch protection rule](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule) in the repository making it require a pull request before merging. We will talk with detail about pull requests later on, but for now you can think about them as specific notifications you will receive when someone is trying to modify the `main` branch. Pull request should be approved before merging in general by other team members.
+
+🙌
+To add a branch protection rule you should go to **Settings > Code and Automation (sidebar) > Branch protection rules** and click in **"Add rule"** in your repository. 
+
+---
+
+# Branch protection rules
+
+
+
+<center>
+
+<img src='/module5/PR-reviews-required-updated.png' width="80%" alt="404 image" />
+
+<center/>
+
+Notes:
+
+🙌
+Under "Branch name pattern", type the branch name or pattern you want to protect. 
+Notice that you can create protect branches different to `main` but we will not cover those cases of use in this tutorial.
+
+---
+
+# Branch protection rules
+
+<center>
+
+<img src='/module5/branches-house-analogy-explained.png' width="80%" alt="404 image" />
+
+<center/>
+
+Notes:
+
+If you think in the analogy of the house, protecting the `main` branch would be similar to avoid creating modifications in the structure of the house meanwhile each member is working in different sections. It is not a good idea to modify the foundation of the house at the same time other team members are working in the walls or windows. If there are changes in the house's foundation there could be problems when combining the parts, including the risk of collapsing the house.
```

Comment:
```suggestion
If you think in the analogy of the house, protecting the `main` branch would be similar to avoid creating modifications in the foundation of the house meanwhile each member is working in different sections. This makes sense since any changes in the house's foundation could give rise to problems when combining the parts, including the risk of collapsing the house.
```

Great to tie it back to this analogy!